croncheck.php

```diff
@@ -292,6 +294,8 @@
             $status == \core\check\result::CRITICAL ||
             $status == \core\check\result::ERROR) {
 
+            $critical = $status == \core\check\result::CRITICAL;
```

Comment:
This logic needs to || itself with the current version of $critical so a later non critical will not reset an earlier critical.

Reply:
oh duh

Reply:
Made it an if (!critical) which is slightly faster to grok on a quick read